masonry/src/widgets/scroll_bar.rs

```diff
@@ -154,7 +154,7 @@ impl Widget for ScrollBar {
                 }
                 ctx.request_render();
             }
-            PointerEvent::PointerUp(_, _) => {
+            PointerEvent::PointerUp(_, _) | PointerEvent::PointerLeave(_) => {
                 self.grab_anchor = None;
                 ctx.request_render();
             }
```

Comment:
At this point I wonder if we should add an `Update::PointerCaptureChanged` event.